docusaurus.config.ts

```diff
@@ -244,6 +244,10 @@ Apache 2.0</a>. Built with &#x2665; and Docusaurus.</p>`,
         respectPrefersColorScheme: true,
       },
     } satisfies Preset.ThemeConfig,
+    customFields: {
+      NODE_ENV: process.env.NODE_ENV,
```

Comment:
I'm not sure if we define the build type anywhere. Is it `production` by default when you do `yarn build`?